package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "eslint-plugin-logdna",
+  "version": "0.0.0",
+  "description": "A collection of ESlint rules for LogDNA",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/logdna/eslint-plugin-logdna.git"
+  },
+  "author": {
+    "name": "LogDNA, Inc.",
+    "email": "help@logdna.com"
```

Comment:
I know this is c&p'd, but I've been questioning this recently - namely the email portion.  I'm not sure who's on the other end of `help@logdna.com`.  Should we exclude the email address?

Reply:
yeah I'd be in favor of dropping the email address (assuming there isn't a compelling reason to have one at all) - communication around these packages is best done through issues anyway

Reply:
removed